test/over_react/component/_deprecated/resize_sensor_test.dart

```diff
@@ -103,7 +103,7 @@ void main() {
         resizes.add(event);
       }, width: width, height: height, onInitialize: onInitialize, resizeSensorProps: resizeSensorProps);
 
-      if (resizeSensorRef.current.props.quickMount) {
+      if (resizeSensorRef!.current!.props.quickMount!) {
```

Comment:
Yeah this is one of the unfortunate cases where defaulted props in class components are a bit less ergonomic, and why for a lot of prop reads within components we'll want to move defaulting out of `defaultProps` so that we won't need `!`. 

That way, we'll get good null-safety without `!`, while also keeping those props as non-required (which non-nullable and `late` are currently planned to imply).